packages/types/Calendar.d.ts

```diff
@@ -72,6 +72,15 @@ export interface AdditionInformation {
   hangoutLink?: string;
 }
 
+export interface RecurringEvent {
+  dtstart?: Date | undefined;
+  interval?: number;
+  count?: number;
+  freq?: RRuleFrequency;
```

Comment:
You're missing an `import type { Frequency as RRuleFrequency } from "rrule";` at the top of the file.